CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.4 (Unreleased)
```

Comment:
This was updated to be one higher than the version in the iot_pnp branch which is where earlier betas were shipped out of.
https://github.com/Azure/azure-sdk-for-c/blob/93fb3362a37753c67aca4b52b852441e097f4dbf/CHANGELOG.md#110-beta4-unreleased

FYI @danewalton, when you do your next sync.